lib/ansible/plugins/connection/ssh.py

```diff
@@ -292,6 +292,7 @@
         description:
             - "Preferred method to use when transferring files over ssh"
             - Setting to 'smart' (default) will try them in order, until one succeeds or they all fail
+            - For OpenSSH >=9.0 you must add an additional option to enable scp (ansible_scp_extra_args="-O")
```

Comment:
the setting is `scp_extra_args`, there is more than one way to add the -O, not just the variable